docs/documentation.html

```diff
@@ -3481,6 +3661,135 @@ <h3>Usecase</h3>
 						</div>
 					</section>
 
+					<!-- Snackbar -->
+					<div id="Snackbar"></div>
+
+					<!-- Snackbar -->
+					<section class="component">
+						<div class="component__info">
+							<div class="component__heading">
+								<h2>Snackbar</h2>
+							</div>
+						</div>
+
+						<div
+							class="component__display--container flex-row"
+						>
+							<button
+								class="btn btn-primary btn__snackbar--leading"
+							>
+								Leading
+							</button>
+							<button
+								class="btn btn-primary btn__snackbar--baseline"
+							>
+								Baseline
+							</button>
+
+							<div
+								class="snackbar__output flex-row"
+							>
+								<div
+									class="snackbar__baseline flex-row"
+								>
+									<div
+										class="snackbar__baseline--text"
+									>
+										Can't send photo.
+										Retry in 5 seconds.
+									</div>
+									<div
+										class="snackbar__baseline--actions"
+									>
+										<a href="#"
+											>Retry</a
+										>
+										<span
+											><button>
+												X
+											</button></span
+										>
+									</div>
+								</div>
+
+								<div
+									class="snackbar__leading flex-row"
+								>
+									<div
+										class="snackbar__leading--text"
+									>
+										Your photo has been
+										archived.
+									</div>
+									<div
+										class="snackbar__leading--actions"
```

Comment:
Why is your code so much towards the right side??

Reply:
it's because of 4 indentations with Prettier